README.md

```diff
@@ -1,30 +1,62 @@
-![GODOT](/logo.png)
+[![GODOT](/logo.png)](https://godotengine.org)
 
-https://godotengine.org
+## Godot Engine
 
-### The Engine
+Homepage: https://godotengine.org
 
-Godot is a fully featured, open source, MIT licensed, game engine. It focuses on having great tools, and a visual oriented workflow that can export to PC, Mobile and Web platforms with no hassle.
-The editor, language and APIs are feature rich, yet simple to learn, allowing you to become productive in a matter of hours.
+#### 2D and 3D cross-platform game engine
 
-### About
+Godot Engine is a feature-packed, cross-platform game engine to create 2D and
+3D games from a unified interface. It provides a comprehensive set of common
+tools, so that users can focus on making games without having to reinvent the
+wheel. Games can be exported in one click to a number of platforms, including
+the major desktop platforms (Linux, Mac OSX, Windows) as well as mobile
+(Android, iOS) and web-based (HTML5) platforms.
 
-Godot has been developed by Juan Linietsky and Ariel Manzur for several years, and was born as an in-house engine, used to publish several work-for-hire titles. Godot is a member project of the [Software Freedom Conservancy](https://sfconservancy.org)
```

Comment:
I'd appreciate if you were less sarcastic in your comments, thanks.

I remove this part not to rip @reduz and @punto- of the credit of their work, but because the new description that comes before it seemed long enough to me that I thought mentioning Godot having started as an in-house engine was maybe not that important in the readme of a source repository. It was useful on Day 1 to give Godot the credibility of a professional engine, but I believe this credibility is established now.

I don't have a strong opinion, but I can of course readd some details about the engine's history.